lib/subwidgets/navbar_widget.dart

```diff
@@ -25,19 +27,27 @@ class _NavbarWidgetState extends State<NavbarWidget> {
         type: BottomNavigationBarType.fixed,
         selectedFontSize: 12.0,
         unselectedFontSize: 12.0,
-        backgroundColor: Colors.purple,
-        selectedItemColor: Colors.orange,
+        backgroundColor: Colors.white,
+        selectedItemColor: onPrimaryColorMaterial.shade400,
+        unselectedItemColor: Colors.black,
         onTap: onTabTapped,
         currentIndex: widget.activePosition,
         items: [
           BottomNavigationBarItem(
-              icon: const Icon(Icons.check_box), label: widget._tasksButton),
+              icon: const Icon(Icons.check_box, size: 30),
```

Comment:
Instead of putting in these measurements directly, can we create a variable for this size and a variable for the add button size and put those in here? It's a constant value, so it should be saved in a constant or member variable somewhere.